src/bot/scripts/card.js

```diff
@@ -728,946 +66,1233 @@ const card = new Command({
+          commandValues.modifier = "l";
+        }
+        if (commandValues.value === undefined) {
+          commandValues.value = 1;
+        }
 
-      if (card.CharacterType !== "Totem") {
+        if (commandValues.name === "%r %r") {
+          commandValues.name =
+            cards.data[Math.floor(Math.random() * cards.data.length)].name;
+          commandValues.modifier = "l";
+          commandValues.value = Math.floor(Math.random() * 7);
+        }
 
-        embed.description += `Max Speed: ${Math.round(card.MaxVelocity * 100) / 100}\n`;
-        embed.description += `Time To Reach Max Speed: ${Math.round(card.TimeToReachMaxVelocity * 100) / 100}\n`;
-        embed.description += `Agro Range Multiplier: ${Math.round(card.AgroRangeMultiplier * 100) / 100}\n\n`;
-      } else {
+        commandValues.value = parseInt(commandValues.value);
 
-        embed.description += "\n";
-      }
+        let highestToDate = 0;
+        let highestCard = null;
 
-      let hasPower = false;
-      for (let field in card) {
+        // card name - similarity
+        for (let card of cards.data) {
+          const lowerArray = card.name.toLowerCase();
 
-        if (field.startsWith("Power") && card[field] !== null) {
+          let sim = similarity(lowerArray, commandValues.name);
 
-          hasPower = true;
-          break;
+          if (sim > highestToDate) {
+            highestToDate = sim;
+            highestCard = card;
+          }
         }
-      }
 
-      // enforcer jimmy aura range
-      if (hasPower || card.Name[0] === "Enforcer Jimmy") {
-
-        embed.description += "**Power Information? - Yes**\n";
-
-        for (let field in stats) {
+        if (highestToDate < 0.4) {
+          return message.channel.send("Card Not Found");
+        }
 
-          if (field === "PowerRange" && stats[field] === 0) {
+        commandValues.matchedCards = highestCard;
+        commandValues.similarity = highestToDate;
 
-            continue;
-          }
+        //const cars = ["ford-lemon",
+        //"lambo-range",
+        //"ferra-pple",
+        //"peug-rapes"];
 
-          if (field.startsWith("Power")) {
+        const url = `https://sppd.feinwaru.com/backgrounds/${commandValues.matchedCards.image}.jpg`;
 
-            embed.description += `${field === "PowerRange" ? field : camelPad(field.slice(5, field.length))}: ${typeof stats[field] === "number" ? Math.round(stats[field] * 100) / 100 : stats[field]}\n`;
-          }
-        }
+        const pathResolve = path.resolve(
+          __dirname,
+          "../assets/cards/art/",
+          commandValues.matchedCards.image + ".jpg"
+        );
 
-        if ( /*card.ChargedPowerRadius !== 0 && */ card.ChargedPowerRegen !== 0) {
+        const directoryPath = path.join(__dirname, "../assets/cards/art/");
+
+        await readDir(directoryPath)
+          .then(async files => {
+            let result = true;
+            for (let downloadfiles of files) {
```

Comment:
these arent downloaded files, thats a very deceiving var name

Reply:
yes, which is what i want